examples/popup.html

```diff
@@ -10,10 +10,8 @@
     <link rel="stylesheet" href="../resources/bootstrap/css/bootstrap-responsive.min.css" type="text/css">
     <style type="text/css">
       .ol-popup {
-        display: none;
```

Comment:
The popup is not visible on load because its position is absolute with a bottom value of 15px: the element is shifted to the bottom of the page. This can be improved by storing the div in an invisible container (https://github.com/openlayers/ol3/blob/master/examples/overlay.html#L68-L74)